Common/Securities/Equity/PatternDayTradingMarginModel.cs

```diff
@@ -0,0 +1,148 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Orders;
+using System;
+
+namespace QuantConnect.Securities.Equity
+{
+    /// <summary>
+    /// Represents a simple margining model where margin/leverage depends on market state (open or close).
+    /// During regular market hours, leverage is 4x, otherwise 2x
+    /// </summary>
+    public class PatternDayTradingMarginModel : SecurityMarginModel
+    {
+        private readonly decimal _openMarketLeverage;
+        private readonly decimal _closedMarketLeverage;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PatternDayTradingMarginModel" />
+        /// </summary>
+        public PatternDayTradingMarginModel()
+            : this(2.0m, 4.0m)
+        {
+        }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PatternDayTradingMarginModel" />
+        /// </summary>
+        /// <param name="closedmarketleverage"></param>
+        /// <param name="openmarketleverage"></param>
```

Comment:
Prefer that we fill out these parameter descriptions -- lots of users and contributors rely on us having full source documentation